netci.groovy

```diff
@@ -470,6 +414,66 @@ branchList.each { branchName ->
     }
 }
 
+// adds steps to a job to download coreclr artifacts and corefx artifacts and invoke run-test.sh.
```

Comment:
I was pointing this to the coreclr repo because I was only expecting coreclr and mscorlib to be debug/checked and not the other corefx packages. If you guys feel we need a debug set of everything from coreclr through all the corefx libraries then I agree it is easier to do that in corefx over coreclr but I'm not sure we need a full debug stack. 

Part of the challenge I worry about here is that the set of tests are very much specific to coreclr and we are working on enabling the test infrastructure for .NET Framework and .NET Native which will not work with this approach. I think it is much easier to do debug testing for the various runtimes in the place we build the runtimes themselves. 

I'm not trying to block there I just want to make sure you guys are thinking about all the testing scenarios we need to handle here.